docs/pages/connect-your-client/web-ui.mdx

```diff
@@ -29,4 +29,55 @@ From the active sessions list, click **Join** and select a participant mode to j
 
 You must have the `join_sessions` allow policy in a role you've been assigned to join sessions in any participant mode. 
 For information about how to configure the `join_sessions` allow policy and participant modes for a role, see 
-[Configure an allow policy](../access-controls/guides/moderated-sessions.mdx#configure-an-allow-policy).
\ No newline at end of file
+[Configure an allow policy](../access-controls/guides/moderated-sessions.mdx#configure-an-allow-policy).
+
+## Idle timeout
+
+After the user logs in, the Teleport Web UI checks every 30 seconds if the session is inactive. If
+so, it logs out the user. A session is considered inactive if more than 10 minutes have passed since
+the user last interacted with any Web UI browser tab, either through keyboard input or mouse
+movement and clicks.
+
+The default idle timeout of 10 minutes can be adjusted in the Auth Service configuration through the
+`web_idle_timeout` setting.
+
+<Tabs>
```

Comment:
The "Connect your Client" docs are intended for end users instead of admins. Is it possible to rephrase the change to target end users? E.g., we could tell them to have an admin change `web_idle_timeout`. 

Ideally, there would be a reference we could link to from here that explains the fields of `cluster_networking_config` for admins, but there doesn't seem to be one. I can open an issue so we can add this.

Reply:
> The "Connect your Client" docs are intended for end users instead of admins. Is it possible to rephrase the change to target end users? E.g., we could tell them to have an admin change `web_idle_timeout`.

Ah, sure, I wasn't aware of that. AFAIK that's the only page dedicated to Web UI, so I didn't even consider putting this anywhere else. I'll adjust the copy a little bit.

> Ideally, there would be a reference we could link to from here that explains the fields of `cluster_networking_config` for admins, but there doesn't seem to be one. I can open an issue so we can add this.

I'd appreciate that, thanks.

Reply:
Added: https://github.com/gravitational/teleport/issues/42233
I've also approved the PR